report/index.html

```diff
@@ -0,0 +1,344 @@
+<!-- Tango Viz - This is a very simple example of using ONLY html+script tags to display a simple
+  graphviz -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>TangoViz</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@allenai/varnish@3.0.5/shellac/shellac.min.css"
+    />
+  </head>
+
+  <body>
+    <div id="root">
+      <noscript>JavaScript is required.</noscript>
+      <!-- Include AI2 header via shellac -->
+      <header>
+        <div class="banner">
+          <div class="content">
+            <a href="https://allenai.org">
+              <img
+                src="https://cdn.jsdelivr.net/npm/@allenai/varnish@3.0.5/shellac/ai2.svg"
+                alt="Allen Institute for AI"
+              />
+            </a>
+          </div>
+        </div>
+        <div class="content header-columns">
+          <!-- TODO: add a logo for tango -->
+          <h2 class="header-title">Tango</h2>
+        </div>
+      </header>
+      <div id="chart"></div>
+    </div>
+
+    <!-- styling is not great, here we do some global css, but keep in mind that the actual viz is
+      an svg, so not all css makes sense -->
+    <style>
```

Comment:
I would be in favor of breaking everything out now --> a CSS file, and separate JS files so that there isn't any JS in script tags within this file. It'd be easier to read for me at least.

Reply:
css pulled out
js soon

Reply:
done